core/src/main/java/bisq/core/dao/governance/bond/lockup/LockupTxService.java

```diff
@@ -103,7 +103,7 @@ private Transaction getLockupTx(Coin lockupAmount, int lockTime, LockupReason lo
             throws InsufficientMoneyException, WalletException, TransactionVerificationException, IOException {
         byte[] opReturnData = BondConsensus.getLockupOpReturnData(lockTime, lockupReason, hash);
         Transaction preparedTx = bsqWalletService.getPreparedLockupTx(lockupAmount);
-        Transaction txWithBtcFee = btcWalletService.completePreparedBsqTx(preparedTx, true, opReturnData);
+        Transaction txWithBtcFee = btcWalletService.completePreparedBsqTx(preparedTx, opReturnData);
```

Comment:
Unrelated to the changes in this PR, but why are we using custom fee for a DAO operation? This is the lockup tx, should probably use the fee from feeService.

Reply:
It is the users responsibility and does not cause harm to others if he set a too low fee and need to resynce/redo the tx. I guess it was set to fee service rate initially but later changed... But yes, there might be arguments pro and con...